src/enums/name-check-error-type.ts

```diff
@@ -0,0 +1,7 @@
+export enum NameCheckErrorType {
+  errorDesignation = 'errorDesignation',
```

Comment:
I think we usually use all-caps for the enum key.

Reply:
will update

Reply:
Thanks.

(The reason we use all-caps is because these are essentially constants, which are traditionally all caps to distinguish them from regular variables.)